frontend/src/components/Breadcrumb.vue

```diff
@@ -108,12 +108,12 @@ export default defineComponent({
       const list: Array<BreadcrumbItem> = [];
       if (environmentSlug) {
         list.push({
-          name: t("common.environment"),
```

Comment:
Did we update all the occurrences of "common.environment"?

Same below.

Reply:
Yes, it is only in BreadCrumb that we need to change to the plural form.

Reply:
PTAL @d-bytebase